### DISCUSSION TITLE
设置项隐藏太深

### DISCUSSION THREAD

**Discussion by odayou** [hiroi-sora/Umi-OCR#491] [Ideas]
大佬你好，非常喜欢你的软件日常使用频率很高
记得好像从新ui开始设置项按大模块进行了拆分，但是现在二级设置隐藏太深了，能否在"全局设置"页内增加进入二级(如OCR截图)的入口呢，作为老用户都会迷茫，新手估计更摸不着北

**Comment by hiroi-sora** [comment]
好滴，收到。我想想怎么优化……
